rust/test_utils/Cargo.toml

```diff
@@ -0,0 +1,7 @@
+[package]
+name = "test_utils"
+version = "0.1.0"
+edition = "2021"
+
+[dependencies]
+run_script = "^0.10.1"
```

Comment:
I am skeptical that this crate is providing sufficient functionality to warrant a fresh dependency.   How about implementing a version that depends only on `std`..  and comparing the two implementations?

Reply:
i will change for std::process::Command